api/app/db/crud/hfi_calc.py

```diff
@@ -1,11 +1,12 @@
 """ CRUD operations relating to HFI Calculator
 """
-from typing import List
+from typing import List, Optional
 from sqlalchemy.engine.cursor import CursorResult
 from sqlalchemy.orm import Session
-from sqlalchemy import desc, insert
+from sqlalchemy import and_, desc, exists, insert, update
```

Comment:
```suggestion
from sqlalchemy import desc, insert, update
```
and_ , exists ? not being used?